Assets/Project/Modules/Enemies/StaticRanged/Scripts/TurretHealth.cs

```diff
@@ -20,16 +23,23 @@ public DamageHitTargetType GetDamageHitTargetType()
     {
         return DamageHitTargetType.Enemy;
     }
+    
 
     public DamageHitResult TakeHitDamage(DamageHit damageHit)
     {
         int receivedDamage = _healthSystem.TakeDamage(damageHit.Damage);
+        
+        
         Debug.Log("youyou");
         if (IsDead())
         {   
-            //TODO: death feedback
+            _enemy.Die();
             Destroy(gameObject);
```

Comment:
Is this destroy here correct?

Reply:
It shouldn't be a Destroy(), but this change will need a complete rework of the turret, adding the mediator. This will take some time, and possibly bugs. I think it's better to keep it as it is for the prototype and change it afterwards